src/javascript/JContent/actions/publishDeletionAction.jsx

```diff
@@ -6,7 +6,8 @@ import {useSelector} from 'react-redux';
 
 const checkAction = node => node.operationsSupport.publication &&
     isMarkedForDeletion(node) &&
-    node.aggregatedPublicationInfo.publicationStatus !== 'NOT_PUBLISHED';
+    (node.aggregatedPublicationInfo.publicationStatus !== 'NOT_PUBLISHED' ||
+        (node.aggregatedPublicationInfo.publicationStatus === 'NOT_PUBLISHED' && node.aggregatedPublicationInfo.existsInLive));
```

Comment:
You should check whether or not existsInLive is undefined here as well.

Reply:
done